### PR TITLE
Added nodejs server + express to the website

### PR DESCRIPTION
**Pull request by gagishmagi**
Hello, please check my changes I Added a server to the project

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const port = 3000
+
+
+app.use(express.static('public'))
+
+
+
+// app.get('/', (req, res) => {
+//   res.send('Hello World!!!!')
+// })
```
Comment (gagishmagi) [inline server.js]: Please remove these comments

**Comment by shirannimni** [comment]
Ok ,i'm on it

File: package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "git_tutorial",
+  "version": "1.0.0",
+  "description": "## TODO LIST in the project",
```
Comment (gagishmagi) [inline package.json]: Maybe put more descriptive description

**Review by skudsi490 — COMMENTED** [review]
Nice work guys

**Review by gagishmagi — COMMENTED** [review]
Please see the notes, fix them and they will be merged to main

File: package.json (hunk 2 above)
Comment (gagishmagi) [reply package.json]: AGMTM

**Pull request merged**